packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss

```diff
@@ -21,18 +21,48 @@
   .ag-rtl {
     .ag-has-focus .ag-cell-focus:not(.ag-cell-range-selected),
     .ag-context-menu-open .ag-cell-focus:not(.ag-cell-range-selected),
-    .ag-has-focus .ag-full-width-row.ag-row-focus .ag-cell-wrapper.ag-row-group,
     .ag-cell-range-single-cell,
     .ag-cell-range-single-cell.ag-cell-range-handle {
-      border: 2px dotted var(--agGrid-color-focus) !important;
-      padding-left: ag-param(cell-horizontal-padding) - 2px;
-      padding-right: ag-param(cell-horizontal-padding) - 2px;
+      border: var(--agGrid-cursor-borderWidth) var(--agGrid-cursor-borderStyle)
```

Comment:
should these tokens have -cursor in them?

Reply:
Yep, these are the focus styles, and we are using the border as focus ring

Reply:
Cursor is the ag-grid/data-grid naming for the focus ring

Reply:
since we can call it anything, I wonder if its ok to name it something that makes more sense to devs without having to know that convention of ag grid

Reply:
Ok, you convinced me :D renamed...